core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -337,14 +337,16 @@ func (r *EvmRegistry) refreshLogTriggerUpkeepsBatch(logTriggerIDs []*big.Int) er
 	for _, id := range logTriggerIDs {
 		logBlock, ok := configSetBlockNumbers[id.String()]
```

Comment:
sid := id.String()

Reply:
we're pushing this also as a hotfix on release, so i would prefer to keep changes minimal for now